backend/s3/s3.go

```diff
@@ -1018,7 +1049,7 @@ func init() {
 				Value:    "sgp1.digitaloceanspaces.com",
 				Help:     "Digital Ocean Spaces Singapore 1",
 				Provider: "DigitalOcean",
-			}, {
+			}, {				
```

Comment:
extra whitespaces

Reply:
@iav thanks, I removed the extra whitespaces.